domains/authorization/Authorization.API/API/Repository/GenericRepository.cs

```diff
@@ -29,6 +29,7 @@ public void Remove(T entity)
     {
         ArgumentNullException.ThrowIfNull(entity);
         Context.Set<T>().Remove(entity);
+        Context.Entry(entity).State = EntityState.Deleted;
```

Comment:
Why is this needed?

Reply:
It's not. removed: https://github.com/Energinet-DataHub/energy-origin/pull/1236/commits/a9609b6404f0a9481a8b3955d075ceaaa11d733b